app/src/main/res/drawable-v24/ripple_click_effect.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Out of curiosity, is there any specific reason for you to use this ripple_click_effect.xml instead of using `android:background="?attr/selectableItemBackground"` ? Will this work for sdk versions < 24? (I'm saying this because of drawable-v24) @jenpaff

Reply:
On testing it with android:background="?attr/selectableItemBackground" it didn't work for me, why would it be more preferable?

Reply:
I think it was only because android:background="?attr/selectableItemBackground" didn't work for me, but if it's the preferably way, I can look into it again! I can test it with a lower sdk if it works.

Reply:
hi ! thanks for pointing that out. I think it was because I was having trouble setting the height just for the background. I updated it to use android:background="?attr/selectableItemBackground" and instead increased the padding to make the effect look bigger and removed the ripple-effect.xml. Let me know if thats okay, here's a screen recording after the changes
![style-ripple-effect3](https://user-images.githubusercontent.com/5339211/57067653-e35c0700-6ccf-11e9-9312-4ef8109b0b87.gif)

Reply:
Thank you @jenpaff for the update! I was just curious. I know the solution you had can help in customizing the effect, but for this case, it wasn't necessary.